src/components/Header/Header.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onSubmit: (title:string) => void;
+  isDisabled: boolean;
+  errorInput: () => void;
+  todos: Todo[];
+  toogleAll: () => void;
+};
+
+export const Header: React.FC<Props> = ({
+  onSubmit,
+  isDisabled,
+  errorInput,
+  todos,
+  toogleAll,
+}) => {
+  const [query, setQuery] = useState('');
+  const handleSubmit = () => {
+    if (!query) {
+      errorInput();
+
+      return;
+    }
+
+    onSubmit(query);
+    setQuery('');
+  };
```

Comment:
```suggestion
  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
   e.preventDefault();
   
    if (!query) {
      errorInput();

      return;
    }

    onSubmit(query);
    setQuery('');
  };
```
and just pass reference to onSubmit